core/domain/question_services_test.py

```diff
@@ -69,6 +69,26 @@ def test_get_question_by_id(self):
             'not found'):
             question_services.get_question_by_id('question_id')
 
+    def test_create_and_get_question_skill_link(self):
+        question_services.create_new_question_skill_link(
+            self.question_id, 'skill_1')
+
+        question_summaries = (
+            question_services.get_question_summaries_linked_to_skills(
+                ['skill_1']))
+        self.assertEqual(len(question_summaries), 1)
+        self.assertEqual(question_summaries[0].id, self.question_id)
+
+    def test_get_question_summaries_by_ids(self):
```

Comment:
I've mentioned this before -- please try to test all possible cases, including cases where summaries aren't in the datastore. Don't just write tests that cover the happy path. Otherwise errors will get exposed for the first time in QA/production, and you really don't want that.

Reply:
Done, checked the None case here and added tests elsewhere.